test/MUXControlsTestApp/App.xaml

```diff
@@ -35,6 +35,153 @@
                 <Setter Property="BorderThickness" Value="1"/>
                 <Setter Property="Padding" Value="8"/>
             </Style>
+
+            <Style TargetType="local:InfoBar" >
+                <Setter Property="Template">
+                    <Setter.Value>
+                        <ControlTemplate TargetType="local:InfoBar">
+                            <Border
+                                x:Name="Container"
+                                Height="70"
+                                Width="1000"
```

Comment:
>Width="1000" [](start = 32, length = 12)

What if your window/panel width is greater than 1000? I think horizontal alignment stretch would be better.